views/eApostilles/uploadFiles.ejs

```diff
@@ -10,79 +14,44 @@
-            data-module="govuk-error-summary">
-            <h2 class="govuk-error-summary__title" id="error-summary-title">
-                There is a problem
-            </h2>
-            <div class="govuk-error-summary__body">
-                <ul class="govuk-list govuk-error-summary__list">
-                    <li>
-                        <a href="#">You can upload a maximum of 50 files.
-                        </a>
-                    </li>
-                </ul>
-            </div>
-        </div>
-    <% } %>
-
-    <% if (infectedFiles.length > 0) { %>
-        <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1"
-            data-module="govuk-error-summary">
-            <h2 class="govuk-error-summary__title" id="error-summary-title">
-                There is a problem
-            </h2>
-            <div class="govuk-error-summary__body">
-                <ul class="govuk-list govuk-error-summary__list">
-                    <li>
-                        <% if (infectedFiles.length === 1) { %>
-                            <a href="#">
-                                The following file cannot be uploaded because it is infected (may contain a virus). You might be able to upload a
-                                different copy of this file:
-                            </a>
-                        <% } else { %>
-                            <a href="#">
-                                The following files cannot be uploaded because they are infected (may contain a virus). You might be able to upload
-                                different copies of these files:
-                            </a>
-                        <% } %>
-                    </li>
-                    <% infectedFiles.forEach((infectedFile) => { %>
-                        <li><%= infectedFile %></li>
-                    <% }); %>
-                </ul>
-            </div>
-        </div>
-    <% } %>
-
-    <% if (noFileUploadedError) { %>
-        <div class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1"
-            data-module="govuk-error-summary">
-            <h2 class="govuk-error-summary__title" id="error-summary-title">
-                There is a problem
-            </h2>
-            <div class="govuk-error-summary__body">
-                <ul class="govuk-list govuk-error-summary__list">
-                    <li>
-                        <a href="#file-upload-form">No files have been selected</a>
-                    </li>
+                    <% if (preUploadErrorsExists) { %>
+                        <li>
+                            <a href="#error-list">One or more files could not be uploaded</a>
+                        </li>
+                    <% } %>
+                    <% if (postUploadErrorsExists) { %>
+                        <% postUploadErrors.forEach((postUploadError) => { %>
```

Comment:
no need to action, more of a code style/cleaniness thing - I would just default to an empty array in `api/controllers/FileUploadController.js`, so you don't have to do length check here. Template files generally include a lot of if statements and can get hard to read